build/build.sh

```diff
@@ -35,7 +35,7 @@ fi
 LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildStatus=${GITSTATUS}"
 
 : ${GITTAG:=$(git describe 2> /dev/null || echo "unknown")}
-LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG}"
+LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG} -X ${REPO_PATH}/pkg/version.minimumSupportedVersion=${MINIMUM_SUPPORTED_VERSION}"
```

Comment:
This variable doesn't seem to exist. This will break standalone/developer builds.

Reply:
Not sure what you mean, this is just pulling in the existing changes from 2.0 which are already in use

Reply:
Sorry, it doesn't break the build. But `minimumSupportedVersion` doesn't exist in 2.1 branch of pkg/version.

Reply:
Good catch, I'll go back and double check then resubmit this PR

Reply:
Hmm, these changes are from January but it looks as if the other PR is also missing from this codebase

Reply:
I'll close this for now, there's much more work to be done